packit_service/worker/testing_farm.py

```diff
@@ -117,18 +144,7 @@ def run_testing_farm_on_all(self):
             ),
         )
 
-    def run_testing_farm(self, chroot: str) -> TaskResults:
-        # inform users about temporarily disabled testing farm
-        self.report_status_to_test_for_chroot(
-            state=CommitStatus.success,
-            description="Testing farm is temporarily disabled.",
-            chroot=chroot,
-            url="https://github.com/packit/packit-service/issues/803",
-        )
-
-        # do not send the request to the testing farm API
-        return TaskResults(success=True, details={})
-
+    def run_testing_farm(self, build_id: int, chroot: str) -> TaskResults:
```

Comment:
I see. 80 lines is IMHO not that huge and I don't see any clear pattern how to split it so I'll leave it as it is for now and might think about it later - this might not be the last TF-related PR.